lib/browser/guest-view-manager.js

```diff
@@ -184,6 +184,32 @@ const attachGuest = function (embedder, elementInstanceId, guestInstanceId, para
     disableBlinkFeatures: params.disableblinkfeatures
   }
 
+  // parse the 'webpreferences' attribute string, if set
+  // this uses the same parsing rules as window.open uses for its features
```

Comment:
> Any suggestions?

After looking into this a bit more, the code for each wouldn't be the exact same since `window.open` handles additional features as well which the `webpreferences` would not, so it might not make sense to have a common function between the two.